nexuse2e-webapp/src/main/frontend/src/assets/i18n/en.json

```diff
@@ -0,0 +1,20 @@
+{
+  "login": {
+    "httpWarning": "You are trying to log in using an insecure connection. Using HTTPS is strongly advised.",
```

Comment:
Should we mention, that the password is send without protection and an attacker can read it in the network traffic?

Reply:
Extended the warning text.